### PR TITLE
Feature/OL-10489 add documentation to tacp modules

### PR DESCRIPTION
**Pull request by xmadsen**
##### SUMMARY
This PR adds in-module documentation to be eventually added to the Ansible documentation site when these modules are submitted to Ansible Core, as well as a README with installation instructions for incorporating the tacp modules from this fork into a user's local Ansible installation.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
tacp_instance
tacp_network
tacp_info

### CODE REVIEW

**Comment by xmadsen** [comment]
Need to rebase this to devel after OL-9822 #9  gets merged in.

**Comment by mariusvigariu** [comment]
Tag it with [WIP] and request a review when its ready for review

File: README.rst [hunk 1]
```diff
@@ -1,115 +1,322 @@
...
+
+- ``~/.ansible/plugins/``
+
+- ``/usr/share/ansible/plugins/``
+
+
+Copy the ``module_utils/tacp_ansible/`` directory from this ansible.lenovo-tacp repo into
+the local ansible installation's ``module_utils/`` directory.
+
+``[user@hostname ansible.lenovo-tacp]# cp -R ./lib/ansible/module_utils/tacp_ansible
```
Comment (patrickjennings) [inline README.rst]: You would probably need to `sudo` here. Did you verify that this works?

Comment (xmadsen) [reply README.rst]: good catch - added sudo

Comment (patrickjennings) [reply README.rst]: I'm still confused. I don't have `/usr/share/ansible/plugins/module_utils` in my Debian setup. Do you have to make this directory? If so, you may need a `sudo mkdir -p /usr/share/ansible/plugins/module_utils`.

File: README.rst [hunk 2]
```diff
@@ -1,115 +1,339 @@
...
+
+- ``~/.ansible/plugins/``
+
+- On CentOS/RHEL systems, ``/usr/share/ansible/plugins/``
+
+|
+
+1. For example, create the ``~/.ansible/plugins/`` directory:
+
+``[user@hostname ansible.lenovo-tacp]# mkdir -p ~/.ansible/plugins/``
```
Comment (patrickjennings) [inline README.rst]: Remove the `[user@hostname ansible.lenovo-tacp]#`

Convention is generally:
```bash
$ mkdir -p ~/.ansible/plugins/
```

File: README.rst [hunk 3]
```diff
@@ -1,115 +1,339 @@
...
+
+2. Copy the ``module_utils/tacp_ansible/`` directory from this ansible.lenovo-tacp repo into
+   the local ansible installation's ``module_utils/`` directory. In this example the module files 
+   are copied to the ``/usr/share/ansible/plugins`` location, which requires ``sudo``.
+
+``[user@hostname ansible.lenovo-tacp]# sudo cp -R ./lib/ansible/module_utils/tacp_ansible
```
Comment (patrickjennings) [inline README.rst]: This should not require sudo.

File: README.rst (hunk 2 above)
Comment (patrickjennings) [reply README.rst]: From the instructions below, you should also make sure `module_utils` and `cloud` is created.

```bash
$ mkdir -p ~/.ansible/plugins/module_utils ~/.ansible/plugins/modules/cloud
```

File: README.rst [hunk 4]
```diff
@@ -1,115 +1,339 @@
...
+   the local ansible installation's ``module_utils/`` directory. In this example the module files 
+   are copied to the ``/usr/share/ansible/plugins`` location, which requires ``sudo``.
+
+``[user@hostname ansible.lenovo-tacp]# sudo cp -R ./lib/ansible/module_utils/tacp_ansible 
+~/.ansible/plugins/module_utils``
+
+3. Copy the ``modules/tacp/`` directory from this ansible.lenovo-tacp repo into
+   the local ansible installation's ``modules/cloud/`` directory.
+
+``[user@hostname ansible.lenovo-tacp]# sudo cp -R ./lib/ansible/modules/tacp
```
Comment (patrickjennings) [inline README.rst]: Remove sudo.

**Review by patrickjennings — CHANGES_REQUESTED** [review]

**Review by patrickjennings — APPROVED** [review]
LGTM

**Pull request merged**